### PR TITLE
Replace string with BigUInt

### PR DESCRIPTION
**Pull request by mariusmihaic**
Replaced balance/value related fields with `BigUInt`, instead of `string` in:
- `ESDT` balance/(s)
- Transaction `value`
- Account `balance`

Additionally:
- Removed `TransactionHash`, which only contained `string hash`
- Added support for `/` operator in `BigUInt`

### CODE REVIEW

File: cli/cli_handler.cpp [hunk 1]
```diff
@@ -83,7 +83,7 @@ void handleIssueESDT(cxxopts::ParseResult const &result)
             esdtProperties
     )->buildSigned(keyFile->getSeed());
 
-    auto const txHash = proxy.send(tx).hash;
+    auto const txHash = proxy.send(tx);
```
Comment (andreibancioiu) [inline cli/cli_handler.cpp:86]: For future - there's also an endpoint that allows one to send multiple transactions: `/transaction/send-multiple` (erdpy uses it, erdjs does not - yet).

File: include/internal/biguint.h [hunk 2]
```diff
@@ -10,7 +10,9 @@ class BigUInt
 
     explicit BigUInt(std::string value);
 
-    BigUInt operator*(BigUInt const &rhs);
+    BigUInt operator*(BigUInt const &rhs) const;
```
Comment (andreibancioiu) [inline include/internal/biguint.h:13]: They do not seem to be used now (sorry if mistaken). But indeed, can be useful to compute, for example, fees (in the future).

**Review by andreibancioiu — APPROVED** [review]

File: cli/cli_handler.cpp (hunk 1 above)
Comment (mariusmihaic) [reply cli/cli_handler.cpp:86]: Good point, could add a task for this, thanks! :+1:

File: include/internal/biguint.h (hunk 2 above)
Comment (mariusmihaic) [reply include/internal/biguint.h:13]: Yes, not used in project for now, but useful for those who wish to use it.

**Review by dragos-rebegea — APPROVED** [review]

**Pull request merged**